### ISSUE TITLE
Cannot find module backo

### ISSUE THREAD

**Issue by todd** [bugsnag/bugsnag-node#125]
### Expected behavior
Bugsnag is unable to be imported by our projects. Appears to be related to the new version just pushed out - reverting to 2.0.1 fixes this issue. Appears that `backo` is a `devDependency` that was not removed from the published version of the package.

### Observed behavior
`bugsnag` is unable to be required by our projects:
```
module.js:471
    throw err;
    ^

Error: Cannot find module 'backo'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/[redacted]/node_modules/bugsnag/lib/sessions/index.js:7:17)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```
### Steps to reproduce
Install the latest version of `bugsnag`. `require('bugsnag')` should fail.

### Version
2.1.0

#### Can't comment on Issues?
Some users have been unable to comment on Github issues when an [adblocker extension is enabled](https://docs.bugsnag.com/platforms/browsers/faq/#is-bugsnag-blocked-by-ad-blockers).
We recommend temporarily disabling the extension, or if that fails, contacting support@bugsnag.com.

**Comment by todd** [comment]
cc @doron2402 @jeffsaildrone

**Comment by bengourley** [comment]
Thanks for the report – this has been fixed and published in 2.1.1. Sorry the inconvenience!
